go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/moby/sys/user v0.3.0
 	github.com/moby/term v0.5.2
 	github.com/nxadm/tail v1.4.11
-	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/ginkgo/v2 v2.23.0
```

Comment:
In view of commit 6df50bec87, this needs to be updated to v2.23.3

(ideally, renovate should be configured to bump ginkgo in both places at once)

Reply:
The alternative is to drop this second copy entirely: https://github.com/containers/podman/pull/25732

Reply:
> (ideally, renovate should be configured to bump ginkgo in both places at once)

It does, but it doesn't work when ginkgo gets updated due transitive updates, i.e. from c/common.